.travis/before_install-linux.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+set -ev
+
+sudo apt-get install -qq pkg-config fuse
+sudo modprobe fuse
+sudo chmod 666 /dev/fuse
+sudo chown root:$USER /etc/fuse.conf
```

Comment:
I haven't seen a noticeable change. The tests take much much longer than the setup.